src/view/home.js

```diff
@@ -92,26 +94,61 @@ const balanceStyles = StyleSheet.create({
     marginTop: 30,
     marginBottom: 5,
   },
-});
+};
 
-const BalanceDisplay = ({
-  depositLabel,
-  channelBalanceLabel,
-  unitLabel,
-  toggleDisplayFiat,
-}) => (
-  <View style={balanceStyles.wrapper}>
-    <Button onPress={toggleDisplayFiat}>
-      <BalanceLabel>
-        <BalanceLabelNumeral>{channelBalanceLabel}</BalanceLabelNumeral>
-        <BalanceLabelUnit>{unitLabel}</BalanceLabelUnit>
-      </BalanceLabel>
-      <H4Text style={balanceStyles.smallLabel}>Chain Deposit</H4Text>
-      <SmallBalanceLabel unit={unitLabel}>{depositLabel}</SmallBalanceLabel>
-    </Button>
-  </View>
+const balanceStyles = createStyles(
+  baseBalanceStyles,
+
+  maxWidth(breakWidth, {
+    wrapper: {
+      width: 350,
+    },
+  }),
+
+  maxWidth(smallBreakWidth, {
+    wrapper: {
+      width: 250,
+    },
+  })
 );
 
+class BalanceDisplay extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      height: 80,
+    };
+  }
+
+  render() {
+    const {
+      depositLabel,
+      channelBalanceLabel,
+      unitLabel,
+      toggleDisplayFiat,
+    } = this.props;
+    return (
+      <View style={balanceStyles.wrapper}>
+        <Button onPress={toggleDisplayFiat}>
+          <BalanceLabel>
+            <BalanceLabelNumeral>{channelBalanceLabel}</BalanceLabelNumeral>
+            <BalanceLabelUnit
+              onLayout={event => {
+                this.setState({ height: event.nativeEvent.layout.height });
+              }}
+              style={{ paddingTop: calculateTopPadding(this.state.height) }}
+            >
```

Comment:
Views should not contain any logic. Is there a reason we need to `setState` here?

Reply:
We need the height of the balance to dynamically calculate the `paddingTop` of the `btc` unit.

Reply:
fixed!